glide-core/src/protobuf/redis_request.proto

```diff
@@ -38,118 +38,553 @@ message Routes {
+    XGroupDestroy                  = 1409;
+    XGroupSetId                    = 1410;
+    XInfoConsumers                 = 1411;
+    XInfoGroups                    = 1412;
+    XInfoStream                    = 1413;
+    XLen                           = 1414;
+    XPending                       = 1415;
+    XRange                         = 1416;
+    XRead                          = 1417;
+    XReadGroup                     = 1418;
+    XRevRange                      = 1419;
+    XSetId                         = 1420;
+    XTrim                          = 1421;
+
+    //// String commands : https://redis.io/commands/?group=string
+
+    Append                         = 1501;
+    Decr                           = 1502;
+    DecrBy                         = 1503;
     /// Type of a get string request.
-    GetString = 2;
-    /// Type of a set string request.
-    SetString = 3;
-    Ping = 4;
-    Info = 5;
-    Del = 6;
-    Select = 7;
-    ConfigGet = 8;
-    ConfigSet = 9;
-    ConfigResetStat = 10;
-    ConfigRewrite = 11;
-    ClientGetName = 12;
-    ClientGetRedir = 13;
-    ClientId = 14;
-    ClientInfo = 15;
-    ClientKill = 16;
-    ClientList = 17;
-    ClientNoEvict = 18;
-    ClientNoTouch = 19;
-    ClientPause = 20;
-    ClientReply = 21;
-    ClientSetInfo = 22;
-    ClientSetName = 23;
-    ClientUnblock = 24;
-    ClientUnpause = 25;
-    Expire = 26;
-    HashSet = 27;
-    HashGet = 28;
-    HashDel = 29;
-    HashExists = 30;
-    MGet=31;
-    MSet=32;
-    Incr=33;
-    IncrBy=34;
-    Decr=35;
-    IncrByFloat=36;
-    DecrBy=37;
-    HashGetAll=38;
-    HashMSet=39;
-    HashMGet=40;
-    HashIncrBy = 41;
-    HashIncrByFloat = 42;
-    LPush = 43;
-    LPop = 44;
-    RPush = 45;
-    RPop = 46;
-    LLen = 47;
-    LRem = 48;
-    LRange = 49;
-    LTrim = 50;
-    SAdd = 51;
-    SRem = 52;
-    SMembers = 53;
-    SCard = 54;
-    PExpireAt = 55;
-    PExpire = 56;
-    ExpireAt = 57;
-    Exists = 58;
-    Unlink = 59;
-    TTL = 60;
-    Zadd = 61;
-    Zrem = 62;
-    Zrange = 63;
-    Zcard = 64;
-    Zcount = 65;
-    ZIncrBy = 66;
-    ZScore = 67;
-    Type = 68;
-    HLen = 69;
-    Echo = 70;
-    ZPopMin = 71;
-    Strlen = 72;
-    Lindex = 73;
-    ZPopMax = 74;
-    XRead = 75;
-    XAdd = 76;
-    XReadGroup = 77;
-    XAck = 78;
-    XTrim = 79;
-    XGroupCreate = 80;
-    XGroupDestroy = 81;
-    HSetNX = 82;
-    SIsMember = 83;
-    Hvals = 84;
-    PTTL = 85;
-    ZRemRangeByRank = 86;
-    Persist = 87;
-    ZRemRangeByScore = 88;
-    Time = 89;
-    Zrank = 90;
-    Rename = 91;
-    DBSize = 92;
-    Brpop = 93;
-    Hkeys = 94;
-    Spop = 95;
-    PfAdd = 96;
-    PfCount = 97;
-    PfMerge = 98;
-    Blpop = 100;
-    LInsert = 101;
-    RPushX = 102;
-    LPushX = 103;
-    ZMScore = 104;
-    ZDiff = 105;
-    ZDiffStore = 106;
-    SetRange = 107;
-    ZRemRangeByLex = 108;
-    ZLexCount = 109;
+    GetString                      = 1504; // Get
```

Comment:
Will rename to `Get` and `Set` if no objections

Reply:
I like the idea. It keeps things more consistent IMO.